incubator/sentry-kubernetes/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels: {{ include "sentry-kubernetes.labels" . | indent 4 }}
+  name: {{ template "sentry-kubernetes.fullname" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "sentry-kubernetes.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "sentry-kubernetes.fullname" . }}
```

Comment:
`{{ template "sentry-kubernetes.serviceAccountName" . }}`

Reply:
whoops!! @unguiculus  thanks for that catch. (overlooked from copying over from closed PR)
fixed it now.

Reply:
forgot to mention that it just happened to work for me because `{{ template "sentry-kubernetes.fullname" . }}` and `{{ template "sentry-kubernetes.serviceAccountName" . }}` evaluated to the same thing. but yeah, we should definitely be using `{{ template "sentry-kubernetes.serviceAccountName" . }}`

Reply:
Please fix it in the deployment as well.

Reply:
@unguiculus great catch. fixed it!